pagopa/io-app
# feat: [#176311366] Added "Send screenshot" checkbox to the add card screen's contextual help funnel

## Short description
When asking for help in the add card screen and choosing "Write to the IO Team", you get the "Attach your screenshot" checkbox.

## List of changes proposed in this pull request
- Checkboxes have been refactored for readability
- It is now possible to conditionally show checkboxes according to the `supportRequestOptions` configuration specified three nodes above

## How to test
- Follow the help funnel from anywhere but the add card screen, you shouldn't see the new checkbox
- Follow the help funnel from the add card screen and see the new checkbox
- Check it and reach Instabug's request modal, see the attached screen
- Also test without checking the checkbox

https://user-images.githubusercontent.com/26572302/106173571-5e4b9d80-6194-11eb-9267-975fb786d65c.mov
